pkgs/servers/gobetween/default.nix

```diff
@@ -0,0 +1,34 @@
+{ buildGo112Module, fetchFromGitHub, lib, enableStatic ? false }:
```

Comment:
This should be `buildGoModule` and assign `buildGo112Module` to `buildGoModule` from within all-packages.nix